spacewalk/setup/bin/spacewalk-setup

```diff
@@ -1163,8 +1163,24 @@ sub setup_services {
   return 1;
 }
 
+sub is_suse_like {
+  my $line;
+  if(! -e '/etc/os-release') {
+    return undef;
+  }
+  open(OS_RELEASE,"/etc/os-release") || die ("Could not open /etc/os-release\n");
```

Comment:
Let's use the open with three arguments for opening files in read-only mode:

http://modernperlbooks.com/mt/2010/04/three-arg-open-migrating-to-modern-perl.html

Reply:
Will change it.